main/src/modules/contextualLayer/planner/include/planner.h

```diff
@@ -0,0 +1,70 @@
+#include "wrdac/clients/icubClient.h"
+
+using namespace std;
+using namespace yarp::os;
+using namespace wysiwyd::wrdac;
+
+class Planner : public RFModule {
+private:
+
+    ICubClient *iCub;
+    double      period;
+    Port        rpc;
+    Port        portToBehavior;
+    Port        toHomeo;
+    Port        getState;
+    BufferedPort<Bottle>        port_behavior_context;
```

Comment:
This might be needed for use with Karma for context of the action. Keeping it there as a comment.